script/source-setup/npm

```diff
@@ -15,3 +15,16 @@ if [ "$1" == "-f" ]; then
 fi
 
 npm install
+
+# setup test fixtures with missing dependencies
+mkdir -p "${BASE_PATH}/test/fixtures/npm-missing"
+cd "${BASE_PATH}/test/fixtures/npm-missing"
+cp ../npm/package.json .
+
+if [ "${1}" == "-f" ]; then
+  find . -not -regex "\.*" -and -not -name "package\.json" -print0 | xargs -0 rm -rf
+fi
+
+npm install
+rm -rf node_modules/glob
```

Comment:
It looks like these fixtures would only be used for a few tests.  Rather than introducing a new fixtures folder, I'd prefer that the test setup copies the npm files to a tmp folder.  Either the test setup or a test case itself could then remove packages to simulate the missing dependency

Reply:
Ok, got it -- I'm creating `missing-dependencies` directory in `npm` to avoid extra fixture, also just copying the required files from the level above.